.travis.yml

```diff
@@ -55,8 +55,8 @@ install:
 - conda config --append channels conda-forge
 - conda config --append channels tofuproject
 - conda info -a
-- conda install -q python="$TRAVIS_PYTHON_VERSION" conda-verify nose nose-timer coverage
-  codecov
+- conda install -q python="$TRAVIS_PYTHON_VERSION" conda-verify coverage codecov
+- pip install pytest
```

Comment:
why not install pytest directly at conda install ? I guess it's because pytest is not available on conda, can you confirm ?

Reply:
yes :)